solana/programs/token-bridge-relayer/src/processor/initialize.rs

```diff
@@ -37,6 +37,15 @@ pub struct Initialize<'info> {
     )]
     pub wormhole_sender: Account<'info, SenderState>,
 
+    #[account(
+        init,
+        payer = owner,
+        space = 8 + RedeemerState::INIT_SPACE,
+        seeds = [RedeemerState::SEED_PREFIX],
+        bump
+    )]
+    pub wormhole_redeemer: Account<'info, RedeemerState>,
```

Comment:
Neither sender, nor redeemer should have a state or require initialization. They are purely used for signing.

Reply:
Done